flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/AbstractQueryableStateTestBase.java

```diff
@@ -492,9 +493,7 @@ public Integer getKey(Tuple2<Integer, Long> value) {
                 jobStatusFuture = clusterClient.getJobStatus(closableJobGraph.getJobId());
             }
 
-            assertEquals(
-                    JobStatus.RUNNING,
-                    jobStatusFuture.get(deadline.timeLeft().toMillis(), TimeUnit.MILLISECONDS));
+            assertThat(jobStatusFuture.get(deadline.timeLeft().toMillis(), TimeUnit.MILLISECONDS)).isEqualTo(JobStatus.RUNNING);
```

Comment:
Is the logic of this test dependent on the deadline, or is it just providing a suitable timeout for `future#get`. If it is the former, this can be done using Asserj's assertion for the future. If it is the latter, please use `FlinkAssertions.assertThatFuture`.

Reply:
This test is mainly to verify the status of the job rather than the deadline, so I tend to use Asserj's assertion for the future